pulsar-broker/src/test/java/org/apache/pulsar/broker/intercept/CounterBrokerInterceptor.java

```diff
@@ -147,6 +147,15 @@ public void beforeSendMessage(Subscription subscription,
                                   Entry entry,
                                   long[] ackSet,
                                   MessageMetadata msgMetadata) {
+        beforeSendMessage(subscription, entry, ackSet, msgMetadata, null);
```

Comment:
We should not change here.
I think the current implementation will introduce a breaking change?
Let's imagine users are using an old interceptor, it will not have this logic.

The expected behavior here is users implemented both 2 method, they will received 2 copies right?